cache/edge_lambdas/src/redirector.ts

```diff
@@ -38,14 +38,30 @@ const paramsAreSubset = (
 
 const filterParams = (
   params: URLSearchParams,
-  allow: Set<string>
+  allow: Set<string>,
+  modify?: {
+    [oldParamName: string]: string;
+  }[]
```

Comment:
nit, there's a helper type for this:
```suggestion
  modify?: Record<string, string>[]
```

Reply:
Additionally - not sure I understand why this is an array, can't it be a single object?

Reply:
It's me future proofing too much, probably won't really ever need to be an array.

Reply:
Might be helpful to clarify what I mean, I see the future-proofing intention here but I can't see a way in which an array would ever be necessary, as this:
```typescript
[
  { oldParam: 'newParam' },
  { anotherOldParam: 'anotherNewParam' },
  { etc: 'etc' }
]
```
could just be one object:
```typescript
{
  oldParam: 'newParam',
  anotherOldParam: 'anotherNewParam',
  etc: 'etc'
}
```

Unless there's an aspect of the desired behaviour I've missed?

Reply:
No you're totally right, I was very much staring at it yesterday knowing that something was wrong and it shouldn't be this complicated but I was too far in. Thanks! 🙏